web/src/lib/components/forms/create-user-form.svelte

```diff
@@ -1,122 +1,121 @@
 <script lang="ts">
-  import { api } from '@api';
-  import { createEventDispatcher } from 'svelte';
```

Comment:
Darn WebStorm

Reply:
I have added documentation for setting up format / lint extensions in VSCode. If you get WebStorm setup you can update the docs `dev-setup.md` if you want so other people using WebStorm can also get instant feedback on lint issues.